exercises/concept/shopping/shopping.jq

```diff
@@ -1,4 +1,4 @@
-# Note, the expressions for tasks 1-4 must end with a comma.
+# Note, the expressions for tasks 1-3 must end with a comma.
```

Comment:
One thing that's not hugely clear to me is what it means to have these multiple expressions in a file. So maybe expand this comment slightly:

```suggestion
# Write each expression under the relevant comment below.
# Expressions are comment-seperated, so the expressions for tasks 1-3 must end with a comma.
```

You can maybe add a third line to actually explain what's going on with the multiple expressions in one file. Without understanding it myself, it's hard to comment on 🙂